nemo/utils/exp_manager.py

```diff
@@ -910,6 +938,24 @@ def configure_loggers(
     trainer._logger_connector.configure_logger(logger_list)
 
 
+class NeMoCheckpointConnector(_CheckpointConnector):
```

Comment:
Where is it used?

Reply:
Use-case is to use the broadcasted checkpoint path from rank 0 (otherwise, the other ranks will not have the s3 checkpoint path as described here: https://github.com/NVIDIA/NeMo/pull/9045#discussion_r1595711273